docs/developer/payments/price-calculations.mdx

```diff
@@ -0,0 +1,138 @@
+#### 5. Adjustment Events:
+  - The `AUTHORIZATION_ADJUSTMENT`, overwrite previous amounts, if an adjustment is present,
+  older authorization events are ignored, and the adjusted amount becomes the new authorized value.
+
+This logic ensures that transaction amounts reflect the current state of the transaction based on its event history while accounting for both processed and pending actions.
+
+### Calculation Details
+
+When the transaction amounts are recalculated, firstly all values are set to zero, 
+then the calculations are applied in the following order:
+  1. recalculate amounts based on events without PSP reference
+  2. recalculate amounts based on `AUTHORIZATION` events
+  3. recalculate amounts based on `CHARGE` events
+  4. recalculate amounts based on `REFUND` events
+  5. recalculate amounts based on `CANCEL` events
+
+#### Authorized Value ([`transactionItem.authorizedAmount`](/api-reference/payments/objects/transaction-item#transactionitemauthorizedamountmoney---))
+  - When an `AUTHORIZATION_SUCCESS` event occurs and there is no `AUTHORIZATION_FAILURE` or the `FAILURE` event is older: value increases by the `SUCCESS` event's amount.
+  - When an `AUTHORIZATION_ADJUSTMENT` event occurs: value is overwritten by the adjustment's amount.
+  - When a `CHARGE_REQUEST` or `CHARGE_SUCCESS` event occurs: value is reduced by the event's amount. 
+  - When a `CANCEL_REQUEST` or `CANCEL_SUCCESS` event occurs: value is reduced by the event's amount. 
+  - The amount cannot be negative, the minimum value is 0.
+
+#### Authorization Pending Value ([`transactionItem.authorizePendingAmount`](/api-reference/payments/objects/transaction-item#transactionitemauthorizependingamountmoney---))
+  - When an `AUTHORIZATION_REQUEST` event occurs and there is no `AUTHORIZATION_FAILURE` or `AUTHORIZATION_SUCCESS` event is: value increases by the `REQUEST` event's amount.
+
+#### Charge Value ([`transactionItem.chargedAmount`](/api-reference/payments/objects/transaction-item#transactionitemchargedamountmoney---))
+  - When an `CHARGE_SUCCESS` event occurs and there is no `CHARGE_FAILURE` or the `FAILURE` event is older: value increases by the `SUCCESS` event's amount.
+  - When a `CHARGE_BACK` event occurs: value decreases by the event's amount.
+  - When a `REFUND_REQUEST` or `REFUND_SUCCESS` event occurs: value is reduced by the event's amount. 
+  - When a `REFUND_REVERSE` event occurs: value is increased by the event's amount. 
+  - The value can be negative.REFUND_REVERSE
+
+#### Charge Pending Value ([`transactionItem.chargePendingAmount`](/api-reference/payments/objects/transaction-item#transactionitemchargependingamountmoney---))
+  - When an `CHARGE_REQUEST` event occurs and there is no `CHARGE_FAILURE` or `CHARGE_SUCCESS` event is: value increases by the `REQUEST` event's amount.
+
+#### Refunded Value ([`transactionItem.refundedAmount`](/api-reference/payments/objects/transaction-item#transactionitemrefundedamountmoney---))
```

Comment:
question: doesn't CHARGEBACK event also increase `refundedAmount`? if not then we can add a note:

```
:::note
Note that `CHARGEBACK` events which work similarly to a refund (decreases chargedAmount value), are not included in **refundedAmount**, since they were not triggered by merchant, but billed by issuing bank.
:::
```

Reply:
Note added